src/services/__tests__/anchor-service.test.ts

```diff
@@ -482,7 +482,7 @@ describe('anchor service', () => {
         ceramicService.putStream(streamId, stream)
       }
 
-      const candidates = await anchorService._findCandidates(requests, 0)
+      const [candidates, _] = await anchorService._findCandidates(requests, 0)
```

Comment:
```suggestion
      const [candidates] = await anchorService._findCandidates(requests, 0)
```

Unless I miss something, there is no need to have temporary "_" variable.